paper-checkbox.html

```diff
@@ -274,7 +274,17 @@
           var inkSize = this.getComputedStyleValue('--calculated-paper-checkbox-ink-size').trim();
           // If unset, compute and set the default `--paper-checkbox-ink-size`.
           if (inkSize === '-1px') {
-            var checkboxSize = parseFloat(this.getComputedStyleValue('--calculated-paper-checkbox-size').trim());
+            var checkboxSizeText = this.getComputedStyleValue('--calculated-paper-checkbox-size').trim();
+
+            // Is the value not in units of 'px'?
```

Comment:
Update: you should just figure out what the unit is, do the same math, then add the same unit back. I think.